Update calling-functions.md (#2705)
enkidevs/curriculum: python/python-core/python-functions/calling-functions.md

Function time!
```

> A parameter in Python is a variable that can be passed into functions so that methods can be performed onto it.

Reviewer: We can do more than methods on any value so I'd maybe remove that part? Also, parameters aren't technically passed in, it's values for those parameters. i.e. the arguments, that are passed in. 

```suggestion
A parameter in Python is a variable that represents a value passed into a function.
```

> In this example, variable `x` is passed into the function as a parameter:

Reviewer: ```suggestion
In this example, variable `x` is defined in the function as a parameter:
```

    print(ans)
```

> To call this function, you must pass a parameter into it. You can pass a number directly or through a variable:

Reviewer: ```suggestion
To call this function, you must pass a value (i.e. an argument) into it, which will put that value into the `x` variable (i.e. parameter) inside the function.

You can pass a number directly or through a variable:
```
